node_modules/oae-core/notifications/css/notifications.css

```diff
@@ -102,7 +107,7 @@
     }
 }
 
-@media (max-width: 480px) {
+@media (max-width: 479px) {
```

Comment:
Positioning is still a bit off on the smallest size:

<img width="446" alt="screen shot 2015-09-22 at 21 43 17" src="https://cloud.githubusercontent.com/assets/109850/10038030/0203aa12-6173-11e5-960e-89218263794a.png">